plugins/node/opentelemetry-plugin-mongodb/test/mongodb.test.ts

```diff
@@ -107,7 +46,7 @@ describe('MongoDBPlugin', () => {
   let client: mongodb.MongoClient;
   let collection: mongodb.Collection;
   const logger = new NoopLogger();
-  const provider = new NodeTracerProvider();
+  const provider = new BasicTracerProvider();
```

Comment:
This is intentional change? Why do we need to switch to `BasicTracerProvider`?

Reply:
I would guess there is no real reason except basic tracer provider is simpler for tests. node tracer provider doesn't do anything needed for tests so the provider used for tests doesn't need to be platform specific

Reply:
It is intentional, i had some trouble because of the PluginLoader automatically loading the mongo plugin for some reason. Since we force the async hooks context manager anyway, its better to use the basic provider to avoid side effect of loading the node one.